cogs/commands/birthday.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Context
+from pytz import timezone, utc
+
+from config import CONFIG
+from models import db_session
+from models.birthday import Birthday as db_Birthday  # avoid name clash
+
+LONG_HELP_TEXT = """
+Hapy birthday!!!!
+
+As is tradtion, wish our dear lord chancellor of the computer a happy birthday
+"""
+
+SHORT_HELP_TEXT = "Hapy birthday!!!!"
+
+
+class Birthday(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        recent = db_session.query(db_Birthday).order_by(db_Birthday.date.desc()).first()
+        if recent:  # store the most recent birthday
+            self.date = recent.date
+            self.age = recent.age
+        else:  # if there is no birthday, initialise
+            self.date = utc.localize(datetime(1970, 1, 1)).astimezone(
+                timezone("Europe/London")
+            )
+            self.age = 0
+
+    @commands.hybrid_command(help=LONG_HELP_TEXT, brief=SHORT_HELP_TEXT)
+    async def birthday(self, ctx: Context):
+        current_date = utc.localize(datetime.now()).astimezone(
+            timezone("Europe/London")
+        )
+        if current_date.date() <= self.date.date():
+            return await ctx.reply(
+                "I'm sorry but our lord chancellor has already been wished a happy birthday today"
```

Comment:
**L**ord **C**hancellor, put some respect on the name 😤

Reply:
"I'm sorry but our high and gracious Lord Chancellor has already been wished a happy birthday today" - this better?